frontend/src/app/containers/App.js

```diff
@@ -36,6 +36,10 @@ class App extends Component {
 
     const experimentsPath = 'experiments/';
 
+    // These are used to expose page state for Optimizely, testing, etc.
```

Comment:
See [here](https://developers.optimizely.com/classic/javascript/code-samples/index.html#conditional-activation-js-variable) (linked to from [the SPA documentation](https://help.optimizely.com/Build_Campaigns_and_Experiments/AngularJS,_Backbone.js,_React,_and_Single_Page_Applications_in_Optimizely_Classic#1._Identify_states_in_your_Single_Page_App)).

In the Optimizely dashboard, you have to configure a set of conditions that will activate the experiment.

Like, if we want to run an A/B test on the Page Shot experiment page for users who do not have Test Pilot installed, the condition would be

```js
window.testPilotPathname === ('/experiments/page-shot') && (!window.testPilotHasAddon)
```

I don't know the technical bits about when Optimizely goes in and checks for the condition. If there's a more appropriate spot/way to accomplish this I'd be happy to implement.

`sendToOptimizely` will tell us when a user converts on the experimentation page. For instance, we can say that whenever `ExperimentDetailsPage Interactions / button click / Install the Add-on` is fired, consider this a successful conversion.

Note: I haven't been able to do testing against Optimizely because of CSP and local dev.

(Aaaand it's merged as I'm typing my long-winded response. Adding it anyway)